.github/pull_request_template.md

```diff
@@ -0,0 +1,32 @@
+## Description
+
+<!-- Write a brief description of this PR. -->
+
+## Related links
+
+<!-- Write the links related to this PR. Private links should be clearly marked as private, for example, '[FOO COMPANY INTERNAL LINK](https://example.com)'. -->
+
+## Notes for reviewers
+
+<!-- Write additional information if necessary. It should be written if there are related PRs that should be merged at the same time. -->
+
+## Pre-review checklist for the PR author
+
+- [ ] I've confirmed the [contribution guidelines](https://github.com/tier4/caret/blob/main/.github/CONTRIBUTING.md).
+- [ ] The PR is ready for review.
+
+## In-review checklist for the PR reviewers
+
+The PR reviewers **must** check the checkboxes below before approval.
+
+- [ ] The PR has been properly tested.
```

Comment:
@takeshi-iwanari  
Which do you think is this checking item is required?

Reply:
@takam5f2 
Thank you for pointing it out.
The check is unnecessary. I remove the line in the new commit.

Reply:
@takeshi-iwanari  
Thank you. I think that the PR is acceptable.